### DISCUSSION TITLE
What are the ways to detect faces of a video?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#818] [Q&A] [answered]
What are the ways to detect faces on a video?

**Comment by pospielov** [comment, marked as answer]
you can split video into the frames and then send them to CompreFace

**Reply by uma-sathish** [reply]
how to upload frames in compreface

**Reply by pospielov** [reply]
You can use our SDK or REST API:
https://github.com/exadel-inc/CompreFace#compreface-sdks
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md
